scripts/build/sign.sh

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
+if ! command -v AzureSignTool &>/dev/null; then
+    echo "AzureSignTool not installed. Signing will be skipped."
+fi
+
```

Comment:
I see this doesn't bail out, but I guess when you call AzureSignTool and it doesn't exist, that bails out anyway.

Reply:
Ah, I meant for this to bail out. I'll fix it